tests/topotests/lib/topotest.py

```diff
@@ -218,12 +215,12 @@ def router_output_cmp(router, cmd, expected):
     )
 
 
-def router_json_cmp(router, cmd, data):
+def router_json_cmp(router, cmd, data, exact=False):
     """
```

Comment:
Might be useful to document `exact` here as well (or point to `json_cmp`).

Reply:
I added some more docu there.